qutip/core/gates.py

```diff
@@ -168,7 +169,7 @@ def ct_gate(*, dtype=None):
     )
 
 
-def rx(phi, *, dtype=None):
+def rx(phi, *, dtype: LayerType = None) -> Qobj:
```

Comment:
```suggestion
def rx(phi: float, *, dtype: LayerType = None) -> Qobj:
```
I guess we should also add types to the other arguments if we're doing that for some gates?

Reply:
Yes, I forgot those. Adding them.